### PR TITLE
Added docs for Auth/Proxy LB configuration

### PR DESCRIPTION
**Pull request by russjones**
Added load balancer configuration information (300 second idle timeout).

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/reference/backends.mdx [hunk 1]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
 
 ## Auth Server State
 
-To run multiple instances of Teleport Auth Server, you must switch to a High Availability secrets back-end first.
+To run multiple instances of Teleport Auth Server, you must switch to a High
+Availability secrets back-end first.
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:46]:
```suggestion
To run multiple instances of Teleport Auth Server, you must switch to a high-availability secrets backend first.
```
- high availability is not a proper noun, so it technically shouldn't be capitalized (though I do see it capitalized elsewhere on this page, so it's probably fine to leave it capitalized here for consistency)
- the rest of the page uses "backend"

File: docs/pages/reference/backends.mdx [hunk 2]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
...
 
-To do this, use a load balancer to create a single auth API access point (AP) and specify this AP in the `auth_server` field of the Teleport configuration file for all Nodes in a cluster. This load balancer should do TCP-level forwarding.
+To do this, use a load balancer to create a single auth API access point (AP)
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:48]: Does "auth API access point" mean the same thing as "a single access point for the Auth Service API"?

File: docs/pages/reference/backends.mdx [hunk 3]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
...
+and specify this AP in the `auth_server` field of the Teleport configuration
+file for all Nodes in a cluster.
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:50]: Does "Nodes" mean "Teleport services"? The [Core Concepts](https://goteleport.com/docs/core-concepts/) page doesn't use the "Nodes" term, and we're moving away from it in the docs.

File: docs/pages/reference/backends.mdx [hunk 4]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
...
 
-**IMPORTANT:** with multiple instances of the auth servers running, special
-attention needs to be paid to keeping their configuration identical. Settings
-like `cluster_name` , `tokens` , `storage` , etc must be the same.
+Configure your load balancer to use Layer 4 (TCP) load balancing with a 300
+second idle timeout.
+
+<Admonition type="tip" title="NOTE">
+With multiple instances of the auth servers running, special attention needs to
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:56]:
```suggestion
With multiple instances of the Auth Service running, special attention needs to
```

File: docs/pages/reference/backends.mdx [hunk 5]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
...
+be paid to keeping their configuration identical. Settings like `cluster_name`,
+`tokens`, `storage`, etc must be the same.
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:58]:
```suggestion
`tokens`, `storage`, etc. must be the same.
```

File: docs/pages/reference/backends.mdx [hunk 6]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
...
 
-<Admonition
-  type="tip"
-  title="NOTE"
->
-  If you terminate TLS with your own certificate at a load
-  balancer you'll need to run Teleport with `--insecure-no-tls`
-</Admonition>
+If using the [default configuration](./networking.mdx), configure your load
+balancer to forward ports `3023`, `3024`, and `3080` to the servers that run
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:66]: Just double-checking: TLS routing isn't enabled by default?

File: docs/pages/reference/backends.mdx [hunk 7]
```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
...
+the Teleport proxy. If you have configured your proxy to use non-default ports,
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:67]:
```suggestion
the Teleport Proxy Service. If you have configured your Proxy Service to use non-default ports,
```

File: docs/pages/reference/backends.mdx [hunk 8]
```diff
@@ -42,53 +42,54 @@ run in a highly available fashion.
...
 ## Auth Server State
 
-To run multiple instances of Teleport Auth Server, you must switch to a High Availability secrets back-end first.
-
-To do this, use a load balancer to create a single auth API access point (AP) and specify this AP in the `auth_server` field of the Teleport configuration file for all Nodes in a cluster. This load balancer should do TCP-level forwarding.
-
-**IMPORTANT:** with multiple instances of the auth servers running, special
-attention needs to be paid to keeping their configuration identical. Settings
-like `cluster_name` , `tokens` , `storage` , etc must be the same.
+To run multiple instances of Teleport Auth Server, you must switch to one of
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:45]:
```suggestion
To run multiple instances of the Teleport Auth Service, you must switch to one of
```

File: docs/pages/reference/backends.mdx [hunk 9]
```diff
@@ -42,53 +42,54 @@ run in a highly available fashion.
...
+the high-availability secrets backend listed below first.
+
+Once you have a high-availability secrets backend and multiple instances of
+Auth Servers running, you'll need to create a load balancer to evenly
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:49]:
```suggestion
the Auth Service running, you'll need to create a load balancer to evenly
```

File: docs/pages/reference/backends.mdx [hunk 10]
```diff
@@ -42,53 +42,54 @@ run in a highly available fashion.
...
+distribute traffic to all Auth Servers instances and have a single point of
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:50]:
```suggestion
distribute traffic to all Auth Service instances and have a single point of
```

File: docs/pages/reference/backends.mdx [hunk 11]
```diff
@@ -42,53 +42,54 @@ run in a highly available fashion.
...
+entry for all components that need to communicate with Auth Server. Use the
```
Comment (ptgott) [inline docs/pages/reference/backends.mdx:51]:
```suggestion
entry for all components that need to communicate with the Auth Service. Use the
```

**Review by ptgott — APPROVED** [review]
Approved with minor wording suggestions to help this correspond with the Core Concepts page.

**Pull request merged**